pom.xml

```diff
@@ -69,41 +69,47 @@
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpclient</artifactId>
-            <version>4.5.1</version>
+            <version>4.5.13</version>
             <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.httpcomponents</groupId>
             <artifactId>httpmime</artifactId>
-            <version>4.5</version>
+            <version>4.5.13</version>
```

Comment:
This dependency is used only to calculate the OSGi package import ranges, please keep using the oldest one that works.